plugin/core/configurations.py

```diff
@@ -38,6 +38,9 @@ def __init__(self, window: sublime.Window, global_configs: dict[str, ClientConfi
     def add_change_listener(self, listener: WindowConfigChangeListener) -> None:
         self._change_listeners.add(listener)
 
+    def get_config(self, name: str) -> ClientConfig | None:
+        return self.all.get(name)
```

Comment:
We do typically use `config_name` nomenclature in other methods.

```suggestion
    def get_config(self, config_name: str) -> ClientConfig | None:
        return self.all.get(config_name)
```